Slack.Automation/Promact.Core.Repository/TaskMailRepository/TaskMailRepository.cs

```diff
@@ -26,8 +28,13 @@ public class TaskMailRepository : ITaskMailRepository
         private readonly IEmailService _emailService;
         private readonly ApplicationUserManager _userManager;
         private readonly IStringConstantRepository _stringConstant;
+
         string questionText = "";
```

Comment:
It is used in two methods and used for storing the question statement. And **questionText** is changable for every request. That why used it as global.
Methods - StartTaskMailAsync and QuestionAndAnswerAsync